### ISSUE TITLE
Generate some inconsistent interfaces

### ISSUE THREAD

**Issue by marticrespi** [dotansimha/graphql-code-generator#701]
**Describe the bug**
Apollo template adds and underscore to some ARGS interfaces declarations, but in all others types (no ARGS) works well, e.g extending Node class.

**To Reproduce**

I have a script in my package.json.

````
  "scripts": {
    "getTypes": "gql-gen --schema https://cms.com/simple/v1/XXXXXXX --header 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1MTgwMTk5NzEsImNsaWVudElkIjoiY2l2Z29zNmNqMDXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX' --template graphql-codegen-apollo-angular-template --out ./server/types.ts"
  },
````
When I run it, it generates all the types and queries and so on, but in some interfaces it declares with an underscore but in their references try to find it without underscore.

**AllArticlesMetaArgs >> _AllArticlesMetaArgs** 

````
  export type AllArticlesMetaResolver<
    R = _QueryMeta,
    Parent = any,
    Context = any
  > = Resolver<R, Parent, Context, AllArticlesMetaArgs>;
  export interface _AllArticlesMetaArgs {
    filter?: ArticleFilter | null;
    orderBy?: ArticleOrderBy | null;
    skip?: number | null;
    after?: string | null;
    before?: string | null;
    first?: number | null;
    last?: number | null;
  }
````
**AllAssetsMetaArgs >> _AllAssetsMetaArgs** 
````
  export type AllAssetsMetaResolver<
    R = _QueryMeta,
    Parent = any,
    Context = any
  > = Resolver<R, Parent, Context, AllAssetsMetaArgs>;
  export interface _AllAssetsMetaArgs {
    filter?: AssetFilter | null;
    orderBy?: AssetOrderBy | null;
    skip?: number | null;
    after?: string | null;
    before?: string | null;
    first?: number | null;
    last?: number | null;
  }
````
**Expected behavior**
Generate the code properly.

**Schema/Documents**
I have uploaded my schema generating it with the other your's template.

https://mega.nz/#!q7wTDKLa!fpl6anSAOm8p_5trKdAt4A5E5C0snBwID54rWQUx1xg

**Environment:**
 - OS: Windows 10
 - Codegen: 0.12.6
 - Node: 8.9.4

**Comment by kamilkisiela** [comment]
@dotansimha that's because we use different logic in `.handlebars` files and `.ts` files. In the template we use [customized toPascalCase](https://github.com/dotansimha/graphql-code-generator/blob/7928031bacdd364ad72a54b31c262e7b5dc5867f/packages/graphql-codegen-compiler/src/handlebars-extensions.ts#L228-L237) but in [the typescript code](https://github.com/dotansimha/graphql-code-generator/blob/7928031bacdd364ad72a54b31c262e7b5dc5867f/packages/templates/typescript/src/helpers/get-field-resolver.ts#L2) it has not that extra stuff.

**Comment by marticrespi** [comment]
Thanks @kamilkisiela @dotansimha for your fast support!

**Comment by dotansimha** [comment]
fixed in 0.13.0. @marticrespi

**Comment by marticrespi** [comment]
Has someone one example how build an apolloServer v2 after generates the types with all the types, resolvers(queries and mutations)?
